jax/tree_util.py

```diff
@@ -306,6 +306,36 @@ class Partial(functools.partial):
 
   (You need to explicitly opt-in to this behavior because we didn't want to give
   functools.partial different semantics than normal function closures.)
+
+  For example, here is a basic usage of ``Partial`` in a manner similar to
+  ``functools.partial``:
+
+  >>> import jax.numpy as jnp
+  >>> add_one = Partial(jnp.add, 1)
+  >>> add_one(2)
+  DeviceArray(3, dtype=int32)
+
+  Pytree compatibility means that the resulting partial function can be passed
+  as an argument within transformed JAX functions, which is not possible with a
+  standard ``functools.partial`` function:
+
+  >>> from jax import jit
+  >>> @jit
+  ... def call_func(f, *args):
+  ...   return f(*args)
+  ...
+  >>> call_func(add_one, 2)
+  DeviceArray(3, dtype=int32)
```

Comment:
This might be a place to mention that for this reason it can make sense just to write `Partial(jnp.add)` sometimes, without any arguments.

Reply:
Done